subprojects/docs/src/samples/user-input/pass-credentials-to-external-tool-via-stdin/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+val login = tasks.register<Exec>("login") {
+    val username: String by project
+    val password: String by project
+    standardInput = java.io.ByteArrayInputStream("$username\n$password".toByteArray())
+    commandLine = listOf("sh", "login.sh")
+}
+
+tasks.register("doAuthenticated") {
+    dependsOn(login)
+    doLast {
+        println("doAuthenticated")
```

Comment:
I think this would look better with a more descriptive message (e.g., "Doing authenticated task")